core/src/test/java/org/apache/iceberg/TestCreateTransaction.java

```diff
@@ -29,61 +31,62 @@
 import org.apache.iceberg.types.TypeUtil;
 import org.apache.iceberg.types.Types;
 import org.assertj.core.api.Assertions;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
 
-@RunWith(Parameterized.class)
+@ExtendWith(ParameterizedTestExtension.class)
```

Comment:
this needs to use `ParameterizedTestExtension` from the iceberg package. Same for `Parameter` / `Parameters`

Reply:
Yes, I am using the same, It might nor be showing with the import . since it's part of same package